test/integration/pairs_test.lua

```diff
@@ -161,7 +165,7 @@ add('test_pairs_no_conditions', function(g)
     t.assert_equals(objects, get_by_ids(customers, {3, 4}))
 
     -- after obj 4 (last)
-    local after = customers[4]
+    local after = crud_utils.flatten(customers[4], g.space_format)
```

Comment:
Seems we need to merge https://github.com/tarantool/crud/pull/43 before

Reply:
Agree.

Reply:
Merged ;)